eva_submission/eload_submission.py

```diff
@@ -27,6 +28,7 @@
     'ena': '18_brokering/ena',
     'scratch': '20_scratch'
 }
+eload_logging_files={}
```

Comment:
You can make this a set rather than a dict, and then in line 73 below just add to the set.
```suggestion
eload_logging_files = set()
```